clients/src/main/java/org/apache/kafka/common/metrics/Sensor.java

```diff
@@ -37,16 +38,20 @@
     private final List<KafkaMetric> metrics;
     private final MetricConfig config;
     private final Time time;
+    private volatile long lastRecordTime;
+    private final long expireInactiveSensorTimeMillis;
```

Comment:
Thanks. Yeah, we should standardise indeed. I'm trying to get us to be more consistent when it comes to naming.